crates/teloxide-core/src/types/inline_keyboard_button.rs

```diff
@@ -71,6 +71,11 @@ pub enum InlineKeyboardButtonKind {
     /// the same chat – good for selecting something from multiple options.
     SwitchInlineQueryCurrentChat(String),
 
+    /// If set, pressing the button will prompt the user to select one of their
+    /// chats of the specified type, open that chat and insert the bot's
+    /// username and the specified inline query in the input field
+    SwitchInlineQueryChosenChat(SwitchInlineQueryChosenChat),
```

Comment:
I understand this is a telegram thing, but what is the difference with `SwitchInlineQuery`?

Reply:
Just a bunch of an optional booleans :)

Reply:
I meant that this is just a superset, you can always use this, right?

Reply:
It looks like that. Maybe the plain `switch_inline_query` will be deprecated in the future in favor of more controllable `switch_inline_query_chosen_chat`, who knows